source/testing/test-helpers.md

```diff
@@ -37,7 +37,7 @@ next one starts.
     is complete.
 * `fillIn(selector, text)`
   - Fills in the selected input with the given text and returns a promise that
-     fulfills when all resulting async behavior is complete. Works with `<select>` elements as well as `<input>` elements.
+     fulfills when all resulting async behavior is complete. Works with `<select>` elements as well as `<input>` elements (with select elements, fill in the value of the option, e.g. 'true' and not the display, e.g. 'Yes').
```

Comment:
This is good information, but perhaps we can reword it to make select less of a special case.  In both scenarios you are setting the `value` of the element, maybe we can just say that instead of calling out `<select>`?